README.md

```diff
@@ -159,6 +160,32 @@ module.exports = {
   ```
 </details>
 
+### esbuild
+
+Current limitations:
+
+- No automatic readable class names in dev
+- The `projectRoot` plugin option must be set to get deterministic class name hashes between build systems
+
+1. Install the dependencies.
+
+```bash
+$ yarn add --dev @vanilla-extract/css @vanilla-extract/esbuild-plugin
+```
+
+2. Add the [esbuild](https://esbuild.github.io/) plugin to your build script.
+
+```js
+const { vanillaExtractPlugin } = require('@vanilla-extract/esbuild-plugin');
+
+require('esbuild').buildSync({
+  entryPoints: ['app.ts'],
+  bundle: true,
+  plugins: [vanillaExtractPlugin({ projectRoot: '...' })],
+  outfile: 'out.js',
+})
+```
```

Comment:
According to [esbuild documentation](https://esbuild.github.io/api/#js-specific-details), we can't use plugins with the synchronous API (`buildSync`) since plugins are asynchronous. So instead of `buildSync`, I think we should use `build` for bundling the code

```suggestion
require('esbuild').build({
  entryPoints: ['app.ts'],
  bundle: true,
  plugins: [vanillaExtractPlugin({ projectRoot: '...' })],
  outfile: 'out.js',
}).catch(() => process.exit(1))
```

Reply:
Ahh good catch. I did know that but forgot when writing the docs.